CHANGES_UNRELEASED.md

```diff
@@ -22,4 +22,14 @@ Use the template below to make assigning a version number during the release cut
 ## [viaduct-reqwest]
 
 ### What's Changed
-  - Update viaduct-reqwest to use reqwest 0.11. ([#4146](https://github.com/mozilla/application-services/pull/4146))
+
+- Update viaduct-reqwest to use reqwest 0.11. ([#4146](https://github.com/mozilla/application-services/pull/4146))
+
+## Logins
+
+### ⚠️ Breaking changes ⚠️
```

Comment:
nit: please put the breaking changes at the *top* of the changelog for easy visibility :-)

Reply:
Sorry, to be more clear: I think we should move the whole "logins" section of the changelog up to the top, so that its breaking changes are the most obvious aspect of the changelog.